### DISCUSSION TITLE
Pie chart error

### DISCUSSION THREAD

**Discussion by vidarusny** [acesseonline/pyreportjasper#104] [Q&A]
Hi,
I contruct report with TIBCO Jaspersoft Studio. Report work fine. So i try to script with pyreportjasper the report update. The reports use many subreports, lot of then have pie charts. And i have error error with sub reports. I try generate alone and my error is : 
`
init: Bootstrapping class not in BootstrapTypesSingleton.getInstance()[class=class org.python.core.PyStringMap]
Traceback (most recent call last):
  File "test.py", line 24, in <module>
    PyReportJasper.process_report()
  File "/usr/local/lib/python3.8/dist-packages/pyreportjasper/pyreportjasper.py", line 182, in process_report
    raise error
NameError: Error fill report: Erro fill internal: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'themeBundle' defined in URL [jar:file:/usr/local/lib/python3.8/dist-packages/pyreportjasper/libs/jasperreports-chart-themes-6.16.0.jar!/net/sf/jasperreports/chartthemes/spring/beans/chartThemesBeans.xml]: Cannot resolve reference to bean 'genericChartTheme' while setting bean property 'themes' with key [TypedStringValue: value [generic], target type [null]]; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'genericChartTheme' defined in URL [jar:file:/usr/local/lib/python3.8/dist-packages/pyreportjasper/libs/jasperreports-chart-themes-6.16.0.jar!/net/sf/jasperreports/chartthemes/spring/beans/chartThemesBeans.xml]: Cannot resolve reference to bean 'titleVisible' while setting bean property 'defaultChartPropertiesMap'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'titleVisible': Initialization of bean failed; nested exception is java.lang.NullPointerException
`

This is the same version of jasper between Studio and pyreportjasper, so i don't understand the problem. Jar files is ok.
Have you some idea ?
Linux mint 20.1
java version : openjdk version "11.0.11" 2021-04-20
python version : 3.8.5

**Comment by jadsonbr** [comment]
All resources used are from the community version ?

**Reply by vidarusny** [reply]
yes.

**Reply by vidarusny** [reply]
The jrxml files were generated under community jaspersoft studio on windows. I destroyed the graphic and recreated under community jasper studio under linux (same version). And it works. an idea ?

**Reply by jadsonbr** [reply]
Create an issue for the problem detailing. If possible, put the execution code and an example report, which we will analyze.
